Content/arm-template.json

```diff
@@ -0,0 +1,50 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "environment": { "type": "string" }
+    },
+    "variables": {
+        "appServicePlan": "[concat('safe-', parameters('environment'), '-web-host')]",
+        "web": "[concat('safe-', parameters('environment'), '-web')]"
+    },
+    "resources": [
+        {
+            "type": "Microsoft.Web/serverfarms",
+            "sku": { "name": "F1" },
+            "name": "[variables('appServicePlan')]",
+            "apiVersion": "2016-09-01",
+            "location": "West Europe"
```

Comment:
I have no experience with ARM but wondering whether all of the options here, like location should be hard-coded?

Reply:
The SKU and Location can both be parameterised (the Location definitely). @theprash The script is already using the `Region` type, which contains consts for all Azure regions, for the `deployment` object so you can use that as a means to supply this as an input parameter to the template. The SKU I don't think is available in code - [this template](https://github.com/Azure/azure-quickstart-templates/blob/master/201-web-app-github-deploy/azuredeploy.json) has an example of showing the different SKUs available and how to set them as input values.

Reply:
I've added a pricingTier parameter for the SKU.